service-descriptions/simple-math-openrpc.json

```diff
@@ -32,6 +32,17 @@
           ],
           "result": { "$ref": "#/components/examples/integerEight" }
         }
+      ],
+      "links": [
+        {
+          "name": "subtractionLink",
```

Comment:
we need to add `name` to meta-schema. when using `links` in practice we will need to be able to reference it by name

Reply:
https://github.com/open-rpc/meta-schema/issues/77